app/src/main/res/values/strings.xml

```diff
@@ -520,4 +520,8 @@ like to connect to:</string>
   <string name="sensor_description_battery_temperature">The current battery temperature</string>
   <string name="search_sensors">Search sensors</string>
   <string name="filter_show_only_enabled_sensors">Only enabled sensors</string>
+  <string name="share_logs">Share logs</string>
+  <string name="share_logs_summary">Collect and share logs with the Home Assistant team will help to solve issues. Please share the logs only if you have been asked to do so by a Home Assistant developer</string>
```

Comment:
Should we mention that sensitive information may be included?

Reply:
Yes you are right. Added a extra alert dialog for that 👍